src/wp-admin/includes/class-rollback-auto-update.php

```diff
@@ -0,0 +1,499 @@
+<?php
+/**
+ * WordPress Plugin Administration API: WP_Rollback_Auto_Update class
+ *
+ * @package WordPress
+ * @subpackage Administration
+ * @since 6.3.0
```

Comment:
I don't deserve props for this, but just want to make note to update the version in all places once this is actually merged.

```suggestion
 * @since 6.4.0
```

Reply:
Is I'm gonna have to fix all the other instances. Not to worry 😉